lib/from_openstudio/geometry/room.rb

```diff
@@ -64,12 +64,166 @@ def self.properties_from_space(space)
+      end
+      # TODO: These are loads assigned to the space directly. How should duplicates created in programtype, if any, be handled? 
+      unless space.people.empty?
+        space.people.each do |people|
+          people_def = people.peopleDefinition
+          # Only translate if people per floor area is specified
+          # Check if schedule exists and is of the correct type
+          if !people_def.peopleperSpaceFloorArea.empty? && !people.numberofPeopleSchedule.empty?
+            sch = people.numberofPeopleSchedule.get
+            if sch.to_ScheduleFixedInterval.is_initialized or sch.to_ScheduleRuleset.is_initialized
+              hash[:people] = Honeybee::PeopleAbridged.from_load(people)
+              break
+            end
+          end
+        end
+      end
+      unless space.lights.empty?
+        space.lights.each do |light|
+          light_def = light.lightsDefinition
+          # Only translate if watts per floor area is specified
+          # Check if schedule exists and is of the correct type
+          if !light_def.wattsperSpaceFloorArea.empty? && !light.schedule.empty?
+            sch = light.schedule.get
+            if sch.to_ScheduleFixedInterval.is_initialized or sch.to_ScheduleRuleset.is_initialized
+              hash[:lighting] = Honeybee::LightingAbridged.from_load(light)
+              break
+            end
+          end
+        end
+      end
+      unless space.electricEquipment.empty?
+        space.electricEquipment.each do |electric_eq|
+          electric_eq_def = electric_eq.electricEquipmentDefinition
+          # Only translate if watts per floor area is specified
+          # Check if schedule exists and is of the correct type
+          if !electric_eq_def.wattsperSpaceFloorArea.empty? && !electric_eq.schedule.empty?
+            sch = electric_eq.schedule.get
+            if sch.to_ScheduleFixedInterval.is_initialized or sch.to_ScheduleRuleset.is_initialized
+              hash[:electric_equipment] = Honeybee::ElectricEquipmentAbridged.from_load(electric_eq)
+              break
+            end
+          end
+        end
+      end
+      unless space.gasEquipment.empty?
+        space.gasEquipment.each do |gas_eq|
+          gas_eq_def = gas_eq.gasEquipmentDefinition
+          # Only translate if watts per floor area is specified
+          # Check if schedule exists and is of the correct type
+          if !gas_eq_def.wattsperSpaceFloorArea.empty? && !gas_eq.schedule.empty?
+            sch = gas_eq.schedule.get
+            if sch.to_ScheduleFixedInterval.is_initialized or sch.to_ScheduleRuleset.is_initialized
+              hash[:gas_equipment] = Honeybee::GasEquipmentAbridged.from_load(gas_eq)
+              break
+            end
+          end
+        end
+      end
+      unless space.otherEquipment.empty?
+        hash[:process_loads] = []
+        space.otherEquipment.each do |other_eq|
+          unless other_eq.designLevel.empty?
```

Comment:
We also need the `sch.to_ScheduleFixedInterval.is_initialized or sch.to_ScheduleRuleset.is_initialized` check here.